libindy/indy-wallet/src/storage/default/transaction.rs

```diff
@@ -14,6 +14,7 @@ impl<'conn> Transaction<'conn> {
             TransactionBehavior::Deferred => "BEGIN DEFERRED",
             TransactionBehavior::Immediate => "BEGIN IMMEDIATE",
             TransactionBehavior::Exclusive => "BEGIN EXCLUSIVE",
+			_ => "BEGIN IMMEDIATE"
         };
```

Comment:
Note sure why this was needed (had to fix a compile error):  https://docs.rs/rusqlite/0.24.2/rusqlite/enum.TransactionBehavior.html

Reply:
I think that's because they added `#[non_exhaustive]` to the enum, although it still just has the three options.